### ISSUE TITLE
benchmark crash Crystal server

### ISSUE THREAD

**Issue by faustinoaq** [kofno/crystal-ide#2]
Hi!
When I do benchmark report the Crystal server crash.

``` crystal
require "benchmark"

Benchmark.bm do |r|
  r.report "p" do
    100.times { p "Ok" }
  end
  r.report "pp" do
    100.times { pp "Ok" }
  end
end

```

![crash](http://i.imgur.com/gPXN5eB.png)
However with other code:
![fine-lint](http://i.imgur.com/AdJCLtQ.png)

**Comment by kofno** [comment]
Are you using the default backend or are you using Scry?

**Comment by faustinoaq** [comment]
I'm using the default backend, I will try with scry :-)
However, The bug is rare, the default crystal backend works well in almost all code, but if I use benchmark then it crash.

**Comment by faustinoaq** [comment]
SCry Works well 😄 🎉 
![scry](http://i.imgur.com/7bNVSCA.png)

[SCry compile log](https://gist.github.com/anonymous/8997db27e81c92167c4ed60e28240842)

**Comment by faustinoaq** [comment]
Hi @kofno :raising_hand_man:
Fixed with #3
